src/obfl-specification.html

```diff
@@ -1805,6 +1847,14 @@ <h4 id="L969" class="include">on-volume-start</h4>
           <dd class="markup note">This attribute is only available in the context of a toc-sequence.</dd>
       </dl>
     </dd>
+  <dt>Variables</dt>
+    <dd><dl>
```

Comment:
We should list here both:
- the variables that can be used in a descendant `evaluate` element
- the variables that can be used in the `use-when` attribute

Reply:
@bertfrees is there a difference? I mean, can you not use all the general variables and also `$started-volume-number` in both a descending `evaluate` and in the `use-when`?

Reply:
Yes, maybe, but:
- We should still be explicit about it. It is currently explained nowhere which variables can be used within `use-when`.
- The "general variables" are not defined anywhere yet. There is only the list in the `evaluate` section.
- Even if in theory (if implemented that way in Dotify) you could the same variables in `use-when` and `evaluate`, it doesn't necessarily make sense to put it that way in the specification. Some variables have no sensible use in a `use-when` so in my opinion it would be better to omit them from the specification. In general I think a good approach would be to be conservative about which variables we include, and only include those for which you can come up with a sensible example.

Reply:
Done.